Application won't boot when using `yarn link`.

Here's the output from `bin/dev`:

```
09:23:02 web.1       | started with pid 68288
09:23:02 worker.1    | started with pid 68289
09:23:02 js.1        | started with pid 68290
09:23:02 light-js.1  | started with pid 68291
09:23:02 light-css.1 | started with pid 68292
09:23:02 light-js.1  | yarn run v1.22.17
09:23:02 js.1        | yarn run v1.22.17
09:23:03 light-js.1  | $ esbuild `bundle exec bin/theme javascript light` --bundle --sourcemap --outdir=app/assets/builds --loader:.png=file --loader:.jpg=file --watch
09:23:03 js.1        | $ node esbuild.config.js --watch
09:23:03 js.1        | ✘ [ERROR] Could not resolve "@bullet-train/fields/app/assets/javascripts/intl-tel-input-utils.js"
09:23:03 js.1        | 
09:23:03 js.1        |     app/javascript/intl-tel-input-utils.js:1:7:
09:23:03 js.1        |       1 │ import "@bullet-train/fields/app/assets/javascripts/intl-tel-input-...
09:23:03 js.1        |         ╵        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
09:23:03 js.1        | 
09:23:03 js.1        |   You can mark the path "@bullet-train/fields/app/assets/javascripts/intl-tel-input-utils.js" as external to exclude it from the bundle, which will remove this error.
09:23:03 js.1        | 
09:23:03 js.1        | ▲ [WARNING] "@charset" must be the first rule in the file
09:23:03 js.1        | 
09:23:03 js.1        |     ../../asdf/local/bullet_train-fields/node_modules/trix/dist/trix.css:300:0:
09:23:03 js.1        |       300 │ @charset "UTF-8";
09:23:03 js.1        |           ╵ ~~~~~~~~
09:23:03 js.1        | 
09:23:03 js.1        |   This rule cannot come before a "@charset" rule
09:23:03 js.1        | 
09:23:03 js.1        |     ../../asdf/local/bullet_train-fields/node_modules/trix/dist/trix.css:6:0:
09:23:03 js.1        |       6 │ trix-editor {
09:23:03 js.1        |         ╵ ^
09:23:03 js.1        | 
09:23:03 js.1        | resolveDir /Users/andrewculver/Sites/clean/bullet_train/app/javascript/controllers
09:23:03 js.1        | path ./**/*_controller.js
09:23:03 js.1        | error Command failed with exit code 1.
09:23:03 js.1        | info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
09:23:03 light-js.1  | [watch] build finished, watching for changes...
09:23:04 js.1        | exited with code 1
09:23:04 system      | sending SIGTERM to all processes
09:23:04 light-css.1 | terminated by SIGTERM
09:23:04 light-js.1  | exited with code 1
09:23:04 worker.1    | terminated by SIGTERM
09:23:04 web.1       | terminated by SIGTERM
```

Here are the steps to reproduce:

Steps to reproduce:

```
export IDE=code # or whatever command you use to open a project in your IDE from the terminal.
git clone git@github.com:bullet-train-co/bullet_train.git bullet_train
cd bullet_train
bin/develop
# Select number 3 and hit <Enter>.
```

Read all the output for context. Basically it automatically links up both the Ruby gem and the npm package to the checked out copy in `local/bullet_train-fields`. 

Open a new terminal:

```
bin/dev
```

That should exit with a failure.

--- a/package.json
+++ b/package.json
@@ -11,9 +11,8 @@
     "app/javascript/**/*.js"
   ],
   "scripts": {
-    "prebuild": "$npm_execpath run clean",
+    "prebuild": "$npm_execpath run clean; $npm_execpath run copy:intl-tel-input-utils",
     "build": "microbundle",
-    "postbuild": "$npm_execpath run copy:intl-tel-input-utils",
     "copy:intl-tel-input-utils": "cp-cli node_modules/intl-tel-input/build/js/utils.js app/assets/javascripts/intl-tel-input-utils.js",
     "clean": "rimraf app/assets/javascripts",
     "prerelease": "$npm_execpath run build",

